src/Cache/Engine/ApcuEngine.php

```diff
@@ -77,18 +76,29 @@ public function write($key, $value)
      * @param string $key Identifier for the data
      * @return mixed The cached data, or false if the data doesn't exist,
      *   has expired, or if there was an error fetching it
+     * @link https://secure.php.net/manual/en/function.apcu-fetch.php
      */
     public function read($key)
     {
         $key = $this->_key($key);
 
         $time = time();
-        $cachetime = (int)apcu_fetch($key . '_expires');
+        $success = false;
+        $cachetime = (int)apcu_fetch($key . '_expires', $success);
+        if ($success === false) {
+            $this->warn(sprintf('Failed to fetch key "%s" from APCu cache.', $key . '_expires'));
```

Comment:
Warning on read failure don't make sense to me. Cache misses are bound to occur, they are not an error case.

Reply:
👍

Reply:
I can easily change this. The idea is that you know when your Cache does not work for whatever reason. Is there a better way?

Reply:
Monitor your cache and application performance to know it is working.
For the app using the cache, this is bullshit!
You don't need no warning on any of these operations.

Reply:
Reads and deletes can and will fail as data expires. Writes failing is more alarming though.